### PR TITLE
Abb202

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
removed

### CODE REVIEW

**Comment by DenisNosnitsin1970** [comment]
the right file is in the rifht directory

**Comment by PietroLiuzzo** [comment]
you are merging here not only this but also changes to ESmy022, perhaps you want someone else to review those?

**Comment by DenisNosnitsin1970** [comment]
Ok, MY 022 went into wrong branch, i did not want it. Please do not approve it for  a while, I will finish MY 022 in this branch, and ask someone else fo a review. once pr has been approved, the dublet abb 202 will be removed.

**Comment by PietroLiuzzo** [comment]
cool, no problem, in fact I have already removed the double file on the server, so, no rush!

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:144]: if you add role="donor" here you do not have to repeat the list below.

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:385]: the whole `<additional>` bit must go above, before the first msPart

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:154]: move `<additional>` from below here

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml]: "only one" or two texts?

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:107]: 1, stub before 1 (replace ":" with ",")

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:361]: I would replace the ref target to p2 with the name of the work, to make the text readable (readings of the Mashafa gessew)

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:340]: I would assume that additions, unless you know something I do not know, belong not to this msPart but to the entire MS and should be moved up, before the first msPart is introduced - they may have been added after the two parts were united

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:385]: ok

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:144]: Should I remove it or it can stay?

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml]: I left Summary empty.

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:107]: ok

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:361]: Ok

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:340]: In this case, the notes belong most probably to part 1, the core.

File: ES/ESmy022.xml
Comment (eu-genia) [reply ES/ESmy022.xml:144]: it can stay, but as a rule, if you assign anywhere in the description roles (donor/owner/scribe) these are read by the system and there is no necessity to repeat them below. no error here.

**Comment by DenisNosnitsin1970** [comment]
ok, thanks.

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:310]: clean `<ref type="work" target="LIT5654MGessew"/>` (you have a second ref inside of target element)

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:405]: i would call this colour "neon pink" (rose does not exist)

File: ES/ESmy022.xml
Comment (eu-genia) [inline ES/ESmy022.xml:404]: no `<date>` element here?

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:404]: I do not understand exactly what is the function of this element. There is no way to date it in any other way as said under "script".

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
you should put the date tag around the words twentieth century and similar - this is called marking up text

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml]: Yes, indeed.

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:310]: Oh, sorry.

File: ES/ESmy022.xml
Comment (DenisNosnitsin1970) [reply ES/ESmy022.xml:404]: Oh, I see. I did, no problem. In this place I did not tag the date before, and we did not tag it here in the catalogue of Dayr as-Suryan

**Pull request merged**
